cardano-node/src/Cardano/Node/TxSubmission.hs

```diff
@@ -94,15 +95,16 @@ submitTx :: ( RunDemo blk
          -> GenTx blk
          -> Tracer IO String
          -> IO ()
-submitTx pInfoConfig nodeId tx tracer =
+submitTx pInfoConfig nodeId tx tracer = do
+    -- TODO: get socketDir from CardanoConfiguration
+    socketDir <- pure "./socket/"
```

Comment:
This seems brittle as the user can now define where they want the socket to live.

Reply:
@Jimbo4350, we need to decide where the socket directory is, somehow.

In `cardano-node` it's now a configurable: https://github.com/input-output-hk/cardano-node/blob/master/cardano-node/src/Cardano/Node/Run.hs#L221

..and I just made `handleTx` respect the configuration as well.